File: 3protected_encapsulation.py
class Human:
    _age = 0;

    def dispalyAge(self) -> None:
        print("Age: ",self._age);

class Person(Human):

    def __init__(self, name, age)->None:
        self.name = name
        self._age = age

    def displayInfo(self):
        print("Name is: ", self.name)
        self.dispalyAge()

p1 = Person("Rakib Khan", 22)
p1.displayInfo()

File: 3protected_encapsulation.cpp
#include<bits/stdc++.h>
using namespace std;

class Human{
    protected:
        int age;

        void displayAge(){
            cout<<"Age: "<<this->age<<endl;
        }
};

class Person : public Human{
    string name;
    public:
        Person(string name, int age){
            this->name = name;
            this->age = age;
        }
        void dispalyInfo(){
            cout<<"Name is: " <<this->name<<endl;
            this->displayAge();
        }
};

int main(){
    
    Person p1("Rakib Khan",22);
    p1.dispalyInfo();
    return 0;
}